### PR TITLE
Sam branch

### PR DESCRIPTION
**Pull request by samlsmith424**
- [X] Commit messages clearly note what has been updated
- [X] Tests are thorough (include Model tests where necessary) and are passing
- [X] The Feature is working on my local server as expected

Complete User Story #8

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class User < ApplicationRecord
-   validates_presence_of :name, :email
+   validates_presence_of :name, :email 
+   validates_uniqueness_of :email, case_sensitive: false
```
Comment (lisataylor5472) [inline app/models/user.rb:3]: I'm very glad to know about validating uniqueness - I would have been lost.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    user = User.new(user_params)
+    if user.save
```
Comment (lisataylor5472) [inline app/controllers/users_controller.rb:8]: Do we need to bulk this up in regards to edge cases?

Comment (samlsmith424) [reply app/controllers/users_controller.rb:8]: That's a great point Lisa! We can definitely keep an eye on this for future tests :)
